recipe_scrapers/hellofresh.py

```diff
@@ -10,8 +13,30 @@ def host(cls, domain="com"):
     def title(self):
         return self.schema.title()
 
+    def cook_time(self):
+        script_tag = self.soup.find("script", {"id": "__NEXT_DATA__"})
+        if script_tag:
+            script_content = script_tag.string
+            total_time_match = re.search(r'"totalTime":"(PT\d+M)"', script_content)
+            if total_time_match:
+                total_time_str = total_time_match.group(1)
+                return get_minutes(total_time_str)
+        return 0
+
+    def prep_time(self):
+        script_tag = self.soup.find("script", {"id": "__NEXT_DATA__"})
+        if script_tag:
+            script_content = script_tag.string
+            prep_time_match = re.search(r'"prepTime":"(PT\d+M)"', script_content)
+            if prep_time_match:
+                prep_time_str = prep_time_match.group(1)
+                return get_minutes(prep_time_str)
+        return 0
+
+    # Note: HelloFresh uses the 'totalTime' metadata field to represent only the cook time.
+    # To get the actual total time, the 'prepTime' and 'totalTime' (which is the cook time) need to be added.
     def total_time(self):
-        return self.schema.total_time()
+        return self.cook_time() + self.prep_time()
```

Comment:
Mostly looks good.  However: I don't think that we should `return 0` as a default fallback from the prep/cook time methods if we can't find timing-related content - instead we should `return None` (meaning: no value found).

And then this would become something like:
```suggestion
        prep_time, cook_time = self.prep_time(), self.cook_time()
        if prep_time or cook_time:
            return (prep_time or 0) + (cook_time or 0)
```